<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdoraUi</class>
 <widget class="QMainWindow" name="AdoraUi">
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Bg_1">
    <property name="geometry">
     <rect>
      <x>-190</x>
      <y>-130</y>
      <width>1911</width>
      <height>1071</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Gui_Materials/B.G/Black_Template.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Gif_1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>651</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/admin/Downloads/img_0119.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Gif_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>9</y>
      <width>701</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Gui_Materials/ExtraGui/initial.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Gif_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>405</y>
      <width>761</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Gui_Materials/ExtraGui/B.G_Template_1.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Gif_4">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>99</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Gui_Materials/ExtraGui/Mr3W.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Bg_2">
    <property name="geometry">
     <rect>
      <x>-16</x>
      <y>160</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/admin/Pictures/Text_input.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Bg_3">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>250</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/admin/Pictures/Text_input.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Bg_4">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>340</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/admin/Pictures/Text_input.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_YouTube">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 0, 0);
font: 12pt &quot;Courier New&quot;;
font: 18pt &quot;Courier&quot;;
color: rgb(0, 255, 0);
</string>
    </property>
    <property name="text">
     <string>YouTube</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Chrome">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 0, 0);
font: 12pt &quot;Courier New&quot;;
font: 18pt &quot;Courier&quot;;
color: rgb(0, 255, 0);
</string>
    </property>
    <property name="text">
     <string>Chrome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Whatsapp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 0, 0);
font: 12pt &quot;Courier New&quot;;
font: 18pt &quot;Courier&quot;;
color: rgb(0, 255, 0);
</string>
    </property>
    <property name="text">
     <string>WhatsApp</string>
    </property>
   </widget>
   <widget class="QLabel" name="Bg_5">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>550</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/admin/Pictures/Text_input.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Bg_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/admin/Pictures/Text_input.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Start">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 0, 0);
font: 12pt &quot;Courier New&quot;;
font: 22pt &quot;Courier&quot;;
color: rgb(0, 255, 0);
</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Exit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>580</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 0, 0);
font: 22pt &quot;Courier New&quot;;
font: 18pt &quot;Courier&quot;;
color: rgb(0, 255, 0);
</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
